virtual_rainforest/models/litter/constants.py

```diff
@@ -54,3 +56,23 @@ class LitterConsts:
     This value was estimated (very unsystematically) from SAFE project data. This
     constant will eventually be removed once the litter is linked to other models.
     """
+
+    cue_metabolic: float = 0.45
+    """Carbon use efficiency of metabolic litter decay [unitless].
+
+    The value given here is taken from :cite:t:`fatichi_mechanistic_2019`, but I can't
+    track down an empirical source.This carbon use efficiency is constant with
+    temperature, soil moisture and substrate stoichiometry. These assumptions are not
+    made in the soil model, but are used for the sake of simplicity here. If an improved
+    version of the litter model gets made, this is a key area to address.
+    """
+
+    cue_structural_above_ground: float = 0.55
+    """Carbon use efficiency of aboveground structural litter decay [unitless].
+
+    The value given here is taken from :cite:t:`fatichi_mechanistic_2019`, but I can't
+    track down an empirical source. This carbon use efficiency is constant with
+    temperature, soil moisture and substrate stoichiometry. These assumptions are not
+    made in the soil model, but are used for the sake of simplicity here. If an improved
+    version of the litter model gets made, this is a key area to address.
```

Comment:
I wouldn't just copy and paste this text for the docstrings.

```suggestion
    The value given here is taken from :cite:t:`fatichi_mechanistic_2019`; see documentation for `cue_metabolic` 
    for details.
```